App Locators/AndroidSiplyLocators.robot

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
+Documentation  Siply Mobile App testing Locators
+Library  AppiumLibrary
+
+
+*** Variables ***
+# Desired Capabilities
+${server}  http://localhost:4723/wd/hub
+${app package}  in.siply.ionic
+${app activity}  com.capitalquotient.siplyv2.MainActivity
+${appium}  appium
+${platform}  Android
+${platform version}  8.1.0
+${device}  ZY32283DJV
+# Language Section
+${siplyLogo}  xpath=/hierarchy/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.FrameLayout/android.widget.FrameLayout/android.widget.FrameLayout/android.view.View/android.view.View/android.view.View/android.view.View/android.view.View/android.view.View/android.view.View[2]/android.widget.ImageView[1]
```

Comment:
Change the xpath and make simple, these xpath seems very much complex. Use Appium Studio for fetching xpath in mobile or UI automator.

Reply:
Ok Will look that.

Reply:
Used relative xpath instead of absolute